### DISCUSSION TITLE
Bot replies without mention

### DISCUSSION THREAD

**Discussion by SpectatorD** [n3d1117/chatgpt-telegram-bot#161] [Q&A] [answered]
there are similar questions, but mine is a bit different
the bot replies to any message, both group and private chat
but I would like it to work only when I mention him or when I reply to his message, tell me how to do it please
I don't know much about coding, I activated it through Powershell
/setinline - enable. and gave administrator rights in the chat

**Comment by n3d1117** [comment]
Hi, the bot does not need administrator rights. Also make sure that privacy mode is enabled (using `/setprivacy` command via BotFather)

**Comment by SpectatorD** [comment, marked as answer]
1. wrote a line
GROUP_TRIGGER_KEYWORD="@botname"
in file .env
2. set the value to activate in (BotFather)
/setprivacy
3. gave admin rights in group chat

**Reply by NeKonnnn** [reply]
Hello! I have the same problem, privacy is enabled and the bot has administrator rights, but it still reacts without mentioning. Any ideas? Tried both @botname in group trigger words and a word. Also, the assistant prompt seems not working, I needed to manually set it via /reset
